iceberg/usecases/injection/src/main/java/com/dremio/iceland/iceberg/injection/DataInjection.java

```diff
@@ -0,0 +1,89 @@
+package com.dremio.iceland.iceberg.injection;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import org.apache.spark.sql.SparkSession;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class DataInjection {
+
+    public static void main(String... args) throws Exception {
+
+        String location = "gdelt.csv";
+
+        if (args.length == 2) {
+            location = args[1];
+        }
+
+        Path file = Paths.get(location);
+        if (!Files.exists(file)) {
+            System.err.println(file.toAbsolutePath() + " doesn't exist");
+            System.exit(1);
+        }
+
+        try (SparkSession spark = SparkProvider.get()) {
+            List<List<String>> lines = parse(file.toFile());
+
+            for (List<String> line : lines) {
+                StringBuilder builder = new StringBuilder();
+                builder.append("INSERT INTO iceland.gdelt.events VALUES(");
```

Comment:
should this be `local` instead of `iceland`?

Reply:
Yeah, we can use `local`, I will fix that.

Reply:
Or what about the catalog name with prefix as type, like `hadoop_catalog`, `nessie_catalog`, etc ?